flashinfer/trace/templates/gemm.py

```
@@ -1072,6 +1072,41 @@ def _fmha_v2_prefill_deepseek_reference(
+    head_dim,
+    seq_len,
+    scale_softmax,
+    scale_bmm1=None,
+    scale_bmm2=None,
+    causal: bool = True,
+    return_lse: bool = False,
+    lse=None,
+    **_unused,
+) -> torch.Tensor:
+    """Reference for FP8 SM120 FMHA v2 standard self-attention."""
+    B, S, H, D = query.shape
```

Ruff (0.16.0)

[warning] Unpacked variable `S` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)